vault/datadog_checks/vault/vault.py

```diff
@@ -29,7 +26,6 @@ class Vault(OpenMetricsBaseCheck):
     SERVICE_CHECK_CONNECT = 'vault.can_connect'
```

Comment:
We probably don't need the line `DEFAULT_API_VERSION = '1'` here since it's now imported from common.py. I saw you left the constant out in the v2 check

Reply:
fixed!